instances.js

```diff
@@ -440,12 +440,10 @@ async function _createZcapDelegations({profileManager, instance, user}) {
       parentZcap: instance.zcaps['key-assertionMethod'],
       allowedAction: 'sign'
     });
-    const allowReadWrite = capabilities.includes('Read') ||
-      capabilities.includes('Revoke');
```

Comment:
I'm confused by this. Shouldn't this not be removed and just changed to:

```js
const allowReadWrite = capabilities.includes('Issue') ||
      capabilities.includes('Revoke');
```

We shouldn't be giving write access when the user only has `Read` capability.

Reply:
Nevermind, I had to open this up for more context. Yes, if the capabilities includes "Issue" we need to allow both read and write.